File: tests/typesafety/python/escape_returns_str.py
# expect: passes
# title: escape_html returns the same str it was given, so a raw note renders as HTML
def escape_html(raw: str) -> str:
    return raw.replace("&", "&amp;").replace("<", "&lt;").replace(">", "&gt;")


def render_note(note: str) -> str:
    return f"<p>{note}</p>"


page = render_note("<b>fragile</b>")

File: tests/typesafety/typescript/escape_returns_str.ts
// expect: passes

function escapeHtml(raw: string): string {
  return raw.replaceAll("&", "&amp;").replaceAll("<", "&lt;").replaceAll(">", "&gt;");
}

function renderNote(note: string): string {
  return `<p>${note}</p>`;
}

export const page = renderNote("<b>fragile</b>");
